src/components/PageBlocks/Configurator.vue

```diff
@@ -80,14 +80,36 @@
           />
         </b-form-group>
 
-        <b-form-checkbox
-          v-model="block.style.wrap.kind"
-          value="card"
-          unchecked-value="plain"
-          switch
+        <b-form-group>
+          <b-form-checkbox
+            v-model="block.style.wrap.kind"
+            value="card"
+            unchecked-value="plain"
+            switch
+          >
+            {{ $t('general.wrap') }}
+          </b-form-checkbox>
+        </b-form-group>
+        <b-form-group
+          :label="$t('general.refresh.label')"
+          :description="$t('general.refresh.description')"
         >
-          {{ $t('general.wrap') }}
-        </b-form-checkbox>
+          <b-input-group>
+            <b-form-input
+              v-model="block.options.refreshRate"
+              class="col-1"
```

Comment:
By cols i meant b-row and b-col, not straight up col class. Look at Number configurator for example